api/v0/runtime.proto

```diff
@@ -151,8 +151,8 @@ message Cell {
   uint64 cpu_shares = 3;
 
   /// Same syntax as the cpus field of this structure, but applies to
-  ///  memory nodes instead of processors.
-  string cpu_mems = 4;
+  /// memory nodes instead of processors.
+  repeated int32 cpu_mems = 4;
```

Comment:
I don't like that we are moving away from the standard format. I think we should provide StandardFormat+. I'll make a more clear comment on my thoughts in a top-level comment